### PR TITLE
Update SEO assets and metadata

### PR DESCRIPTION
**Pull request by Jennievon**
### Why this change is needed

Please provide a description and a link to the underlying ticket

https://github.com/ten-protocol/ten-internal/issues/3848

### What changes were made as part of this PR

Please provide a high-level list of the changes made

this update aims to refresh marketing materials and improve page SEO
- add new promotional images for bridge, gateway & Tenscan tools
- revise SEO metadata across tools/platforms to enhance discoverability

### PR checks pre-merging

Please indicate below by ticking the checkbox that you have read and performed the required
[PR checks](https://github.com/ten-protocol/ten-internal/blob/main/dev-ops-docs/dev-pr-checks.md)

- [ ] PR checks reviewed and performed

### CODE REVIEW

File: tools/tenscan/frontend/src/lib/siteMetadata.ts [hunk 1]
```diff
@@ -2,15 +2,65 @@ import { socialLinks } from "./constants";
...
+  metaTitle: "Tenscan - Real-Time Blockchain Explorer",
   description:
-    "Tenscan allows you to explore and search activities taking place on the Ten Layer 2 blockchain",
-  siteUrl: "https://ten.xyz",
-  siteLogo: ``,
-  siteLogoSquare: ``,
-  email: "team@obscu.ro",
+    "The privacy-preserving blockchain explorer for TEN. View transaction history and smart contract interactions without compromising sensitive data. Experience the future of blockchain transparency.",
+  keywords:
+    "blockchain explorer, real-time blockchain data, dapps, l2, encryption, layer2, crypto transactions, blockchain analysis, Tenscan, block explorer, TEN network, TEN blockchain, TEN explorer",
```
Comment (CaisManai) [inline tools/tenscan/frontend/src/lib/siteMetadata.ts:9]: lets add TEN, TEN Protocol, TEN chain as keywords too

File: tools/tenscan/frontend/src/lib/siteMetadata.ts [hunk 2]
```diff
@@ -2,15 +2,65 @@ import { socialLinks } from "./constants";
...
-  metaTitle:
-    "Ten, a decentralized Layer 2 Rollup protocol designed to hyper-scale and encrypt the Ethereum blockchain.",
+  metaTitle: "Tenscan - Real-Time Blockchain Explorer",
   description:
-    "Tenscan allows you to explore and search activities taking place on the Ten Layer 2 blockchain",
-  siteUrl: "https://ten.xyz",
-  siteLogo: ``,
-  siteLogoSquare: ``,
-  email: "team@obscu.ro",
+    "The privacy-preserving blockchain explorer for TEN. View transaction history and smart contract interactions without compromising sensitive data. Experience the future of blockchain transparency.",
```
Comment (CaisManai) [inline tools/tenscan/frontend/src/lib/siteMetadata.ts:7]: We want to stay away from the words privacy/private - can we change "The privacy-preserving blockchain explorer for TEN" to "TenScan allows you to explore and search the TEN blockchain"

**Review by CaisManai — CHANGES_REQUESTED** [review]

File: tools/tenscan/frontend/src/lib/siteMetadata.ts (hunk 2 above)
Comment (Jennievon) [reply tools/tenscan/frontend/src/lib/siteMetadata.ts:7]: Maybe we update this too, @CaisManai? https://github.com/ten-protocol/go-ten/blob/56fee0ee2e862f2c6d5b978a2f1956ec58ff80b3/design/security/Source_of_truth.md?plain=1#L48

Comment (Jennievon) [reply tools/tenscan/frontend/src/lib/siteMetadata.ts:7]: Also, is it `Tenscan` or `TenScan`? @CaisManai

Comment (0xjba) [reply tools/tenscan/frontend/src/lib/siteMetadata.ts]: TENScan, I believe

**Review by 0xjba — APPROVED** [review]
LGTM

**Review by CaisManai — APPROVED** [review]

**Pull request merged**